lib/validation.py

```diff
@@ -65,6 +65,7 @@ class Attack(Effect):
     hit: List[Effect]
     miss: List[Effect]
     attackBonus: Optional[str255]
+    adv: Optional[Literal[-1, 0, 1, 2]]
```

Comment:
Should this be `Optional[str255]` since `adv` is allowed to be an IntExpression?

Reply:
Fun fact! I changed that locally but apparently never pushed that commit!